src/services/AddressService.test.js

```diff
@@ -0,0 +1,50 @@
+import { getFullAddress } from "./AddressService";
```

Comment:
Exported functions should be UpperCase, not blocking, will updated in another pr.